Build with MSVC 12 failed with error C2220: warning treated as error - no 'object' file generated

When building with MSVC 12 (VisualStudio 2013) the build fails with the error message:

> error C2220: warning treated as error - no 'object' file generated    cpputests\CppUTestExt\MockSupport_cTestCFile.c	111	1	CppUTestExtTests

The reason are the following warnings that are treated as errors:

> warning C4244: 'function' : conversion from 'double' to 'unsigned int', possible loss of data   cpputest\tests\CppUTestExt\MockSupport_cTestCFile.c	103	1	CppUTestExtTests
> warning C4244: 'function' : conversion from 'double' to 'long', possible loss of data    cpputest\tests\CppUTestExt\MockSupport_cTestCFile.c	107	1	CppUTestExtTests
> warning C4244: 'function' : conversion from 'double' to 'unsigned long', possible loss of data    cpputest\tests\CppUTestExt\MockSupport_cTestCFile.c	111	1	CppUTestExtTests

--- a/tests/CppUTestExt/MockSupport_cTestCFile.c
+++ b/tests/CppUTestExt/MockSupport_cTestCFile.c
@@ -100,15 +100,15 @@ void all_mock_support_c_calls(void)
     mock_c()->intReturnValue();
     mock_c()->returnValue();
 
-    mock_c()->expectOneCall("boo2")->andReturnUnsignedIntValue(1.0);
+    mock_c()->expectOneCall("boo2")->andReturnUnsignedIntValue(1);
     mock_c()->actualCall("boo2")->unsignedIntReturnValue();
     mock_c()->unsignedIntReturnValue();
 
-    mock_c()->expectOneCall("boo3")->andReturnLongIntValue(1.0);
+    mock_c()->expectOneCall("boo3")->andReturnLongIntValue(1);
     mock_c()->actualCall("boo3")->longIntReturnValue();
     mock_c()->longIntReturnValue();
 
-    mock_c()->expectOneCall("boo3")->andReturnUnsignedLongIntValue(1.0);
+    mock_c()->expectOneCall("boo3")->andReturnUnsignedLongIntValue(1);
     mock_c()->actualCall("boo3")->unsignedLongIntReturnValue();
     mock_c()->unsignedLongIntReturnValue();
 

